src/tests/ftest/list_tests.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python2 -u
```

Comment:
Aren't we supposed to stop using python2?

Simply using launch.py works on master.  Why does this need separating here?

Reply:
We created this script because the executor nodes do not fully support python3.  launch needs python3-cluster-shell and python3-avocado; clustershell was updated but python3-avocado is not installed(DCO-9721). This script does not get packaged in the rpms.  So nothing with python2 will shipped and unit/functional tests use python3.  It is just this one instance were python2 is still used to get the list of tests to run.